SampleAndroid/app/src/main/java/com/vipaar/lime/samplehl/Utils.java

```diff
@@ -0,0 +1,108 @@
+package com.vipaar.lime.samplehl;
+
+import com.vipaar.lime.hlsdk.HLTheme;
+import com.vipaar.lime.hlsdk.client.HLCall;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class Utils {
+
+    public static HLCall getSessionData(JSONObject json, boolean autoEnableCamera, boolean autoEnableMic, boolean enableMiniview)
+      throws JSONException {
+        return new HLCall(
+          json.getString("session_id"),
+          json.getString("session_token"),
+          json.getString("user_token"),
+          json.getString("url"),
+          BuildConfig.GALDR_API_KEY,
+          "Jason",
+          "https://www.securenvoy.com/sites/default/files/legacy-uploads/2013/10/pizza_hut_logo.jpg",
+          autoEnableCamera,
+          autoEnableMic,
+          false,
+          enableMiniview);
+    }
+
+    public static HLCall renewSessionData(HLCall call, boolean autoEnableCamera, boolean autoEnableMic, boolean enableMiniview) {
+        return new HLCall(
+          call.getSessionId(),
+          call.getSessionToken(),
+          call.getUserToken(),
+          call.getGssUrl(),
+          BuildConfig.GALDR_API_KEY,
+          "Jason",
+          "https://www.securenvoy.com/sites/default/files/legacy-uploads/2013/10/pizza_hut_logo.jpg",
+          autoEnableCamera,
+          autoEnableMic,
+          false,
+          enableMiniview);
+    }
+
+    public static HLTheme createTheme() {
```

Comment:
Is this still valid? I thought we didn't have themes in the new UI?

Reply:
I never felt like we for sure for sure dropped support.